database/factories/CustomFieldFactory.php

```diff
@@ -26,6 +26,7 @@ public function definition()
             'format' => '',
             'element' => 'text',
             'auto_add_to_fieldsets' => '0',
+            'show_in_requestable_list' => '0',
```

Comment:
Did this leak in from your work on the 'show in requestable' feature?

Reply:
Yes - since I was messing with the seeders anyway, I figured I'd add that in as well.